.github/ISSUE_TEMPLATE/bug-template.md

```diff
@@ -0,0 +1,25 @@
+Thank you for taking the time to let us know about the issue you found. Please provide us with the information below and we will look at it as soon as we are able.
+
+## Description
+
+*Please provide a short description of the bug*
+
+## Action Taken
+
+*In what way were you interacting with the application when you discovered the issue?*
```

Comment:
@carltonsmith i love this! can we have some examples here? uploading, downloading/viewing data, requesting access, etc. also, in case multiple issues are experienced, should we make room for that?

Reply:
I think examples would be good for an ADR or something of that sort, but not necessarily for a template. There is a tendency to leave that kind of thing in the submission which can make it confusing to read. Thoughts?

Reply:
I do think I can be more clear, though.

Reply:
How is this?